packages/flutter/lib/src/widgets/basic.dart

```diff
@@ -4799,7 +4827,7 @@ class Flex extends MultiChildRenderObjectWidget {
 ///    that may by sized smaller (leaving some remaining room unused).
 ///  * [Spacer], a widget that takes up space proportional to its flex value.
 ///  * The [catalog of layout widgets](https://flutter.dev/widgets/layout/).
-class Row extends Flex {
+class Row extends StatelessWidget with MultiChildRenderObjectWidgetMixin, FlexMixin {
```

Comment:
Why MultiChildRenderObjectWidgetMixin is needed?

Reply:
Removed